Pyspark/regular_patients/regular_patients.py

```diff
@@ -0,0 +1,106 @@
+'''This module contains the class PatientsData to modelate 
+monthly patient information using Dataframe API from Pyspark'''
+from pyspark.sql import DataFrame
+from pyspark.sql import SparkSession, DataFrameReader
+from pyspark.sql.functions import col, lit, lpad, to_date, lag, months_between, when
```

Comment:
you can just replace this with import `pyspark.sql.functions as f `and use all function like `f.col()`